app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/CheckShopExistenceForm.kt

```diff
@@ -0,0 +1,17 @@
+package de.westnordost.streetcomplete.quests.existence
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AbstractOsmQuestForm
+import de.westnordost.streetcomplete.quests.AnswerItem
+
+class CheckShopExistenceForm : AbstractOsmQuestForm<Unit>() {
+    override val buttonPanelAnswers get() =
+        listOf(
+            AnswerItem(R.string.quest_generic_hasFeature_no) {
+                replaceShop()
+            },
+            AnswerItem(R.string.quest_generic_hasFeature_yes) {
+                applyAnswer(Unit)
+            }
+        )
```

Comment:
This can be simplified even more: No need for a getter, and reformatting to single lines to improve readability (this is subjective of course).

```suggestion
    override val buttonPanelAnswers = listOf(
        AnswerItem(R.string.quest_generic_hasFeature_no) { replaceShop() },
        AnswerItem(R.string.quest_generic_hasFeature_yes) { applyAnswer(Unit) },
    )
```